third_party/s2n-bignum/include/s2n-bignum_aws-lc.h

```diff
@@ -63,6 +63,8 @@ static inline void p256_montjscalarmul_selector(uint64_t res[S2N_BIGNUM_STATIC 1
   else { p256_montjscalarmul(res, scalar, point); }
 }
 
+extern void bignum_montinv_p256(uint64_t z[S2N_BIGNUM_STATIC 4], const uint64_t x[S2N_BIGNUM_STATIC 4]);
```

Comment:
Can we document what the function does and that it's constant-time?

Reply:
done.